models/Follow.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types
```

Comment:
Are you sure `ObjectId` has the right value? 

Should you not have written `const ObjectId = mongoose.Schema.Types.ObjectId` instead?